### DISCUSSION TITLE
[Q&A] GLPI Agent 1.10 on MS-Windows, configuration information into Registry ?

### DISCUSSION THREAD

**Discussion by homebrewtainy** [glpi-project/glpi-agent#715] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hy @g-bougard,

On a fresh installation of GLPI Agent 1.10 on MS-Windows (11 Pro 23H2) , what are the configuration values ​​and locations that allow the Collect and/or Deploy modules to be activated or not?

On a two-phase installation, by launching the MSI from Explorer (without the two modules concerned), then from the Administrator CMD (with the modules) I do not see any variation in the configuration in the Windows Registry:

 - KLM\SOFTWARE\GLPI-Agent
 - KLM\SOFTWARE\GLPI-Agent\Installer

Clearly not a bug but...:
Magic! Or twisted and superfluous question? :-)

**Comment by g-bougard** [comment, marked as answer]
Hi @homebrewtainy 

you need to use ADDINSTALL= parameter during installation. This is documented here:
https://glpi-agent.readthedocs.io/en/latest/installation/windows-command-line.html#command-line-parameters

**Reply by homebrewtainy** [reply]
Thanks for the clarification.

**Reply by danix1** [reply]
Parameter is ADDLOCAL. example: ADDLOCAL=feat_AGENT,feat_DEPLOY
